admin/services/check-form.service.js

```diff
@@ -332,6 +332,23 @@ const checkFormService = {
 
   removeWindowAssignment: async (formId, windowId) => {
     return checkFormDataService.sqlRemoveWindowAssignment(formId, windowId)
+  },
+
+  getParsedCheckForms: async (batchIds) => {
+    if (!batchIds || !Array.isArray((batchIds))) {
+      throw new Error('batchIds list empty or not defined')
+    }
+    let checkForms
+    try {
+      checkForms = await checkFormDataService.sqlFindByIds(batchIds)
+      checkForms = checkForms.map(cf => {
+        cf.formData = JSON.parse(cf.formData)
+        return cf
+      })
+    } catch (error) {
+      throw new Error(error)
```

Comment:
I don't understand this.  You catch an error only to re-throw it?  Why?

Reply:
@jon-shipley was redundant and now removed as it doesnt really serve any purpose.